backend/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

The boilerplate `test` script is not needed, we can add a script to run the server instead.
```suggestion
    "serve": "nodemon index.js"
```

---

We can also remove this script from the root `package.json`
```
    "server": "python backend/manage.py runserver"
```